### PR TITLE
Fix validate on integration with no command

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29429

## Description
Also added validation that the integration has something that is runnable

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 1]
```diff
@@ -842,3 +845,25 @@ def is_valid_description(self, beta_integration: bool = False) -> bool:
...
+        Returns:
+            if there's at least one runnable in the yaml
+        """
+        script = self.current_file.get('script', {})
+
+        if not any([
+            script.get('commands'), script.get('isfetch', script.get('isFetch')), script.get("feed"), script.get('longRunning')]
+        ):
+            self.is_valid = False
+            print_error(f"{self.file_path}: Could not find any runnable command in the integration."
```
Comment (roysagi) [inline demisto_sdk/commands/common/hook_validations/integration.py:866]: do not use `print_error`, create a new error and error-code using the as seen in errors.py and call it via `handle_error()`

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 2]
```diff
@@ -842,3 +845,25 @@ def is_valid_description(self, beta_integration: bool = False) -> bool:
...
+            feed
+            long-running
+
+        Returns:
+            if there's at least one runnable in the yaml
+        """
+        script = self.current_file.get('script', {})
+
+        if not any([
+            script.get('commands'), script.get('isfetch', script.get('isFetch')), script.get("feed"), script.get('longRunning')]
```
Comment (Itay4) [inline demisto_sdk/commands/common/hook_validations/integration.py:863]: `script.get('isfetch', script.get('isFetch'))` ?

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 3]
```diff
@@ -234,6 +235,8 @@ def is_valid_default_arguments(self):
             bool. Whether a reputation command hold a valid argument
         """
         commands = self.current_file.get('script', {}).get('commands', [])
+        if commands is None:  # If no commands, at least isFetch should be.
```
Comment (Itay4) [inline demisto_sdk/commands/common/hook_validations/integration.py:238]: not sure i understand how that comment is related here

**Comment by Itay4** [comment]
please update changelog

**Review by Itay4 — APPROVED** [review]

**Pull request merged**